tests/EvmLookupServer/Test_EvmLookupServer.cpp

```diff
@@ -650,7 +650,8 @@ BOOST_AUTO_TEST_CASE(test_eth_get_balance) {
                     "0xe8d4a51000");
 }
 
-BOOST_AUTO_TEST_CASE(test_eth_get_block_by_number) {
+BOOST_AUTO_TEST_CASE(test_eth_get_block_by_number,
```

Comment:
tests disabled until a later PR - the TXs now need to be well formed/signed and we know the RLP in the case they are Eth TXs - which is going to be a little time consuming.